core/modules/system/system.menu.inc

```diff
@@ -27,6 +27,7 @@ function system_menu_block_defaults($menu_name) {
     'expand_all' => FALSE,
     'clickdown' => FALSE,
     'collapse' => 'default',
+    'toggle_text' => t('Menu'),
```

Comment:
I don't think we want to store the translated string here. It needs to be just 'Menu' so it gets checked in the theme function where you also have it wrapped in `t()`.

Reply:
Good catch.